start.py

```diff
@@ -536,40 +543,57 @@ def _command_condition(deployment, command_id, command_description):
     return not active and success
 
 
-# TODO: Get rid of the excess amount of code duplication between the following
-# function and _configure_kudu.
-def _configure_kafka(deployment, cluster, kafka_version):
+def _deactivate_service_parcel(deployment, cluster, product, version):
+    parcel_found_state = 'not_found'
```

Comment:
It seems weird to have these status strings passed back and forth... Shouldn't the calling function be able to tell the situation in the cluster just by querying the CM API?

Reply:
Done.